lib/learn_to_rank/data_pipeline/bigquery.rb

```diff
@@ -4,7 +4,7 @@ module LearnToRank::DataPipeline
   module Bigquery
     def self.fetch(credentials)
       now = Time.now
-      before = now - 2 * 7 * 24 * 60 * 60
+      before = now - 3 * 30 * 24 * 60 * 60
```

Comment:
I think `90 * 24 * 60 * 60` would be marginally clearer here.

Reply:
Thanks, I've made that change!